### DISCUSSION TITLE
What the meaning of Block-std package

### DISCUSSION THREAD

**Discussion by nhattran998** [toeverything/blocksuite#4810] [Q&A]
Hi team, I currently want to build my own editor base on @blocksuite/store! After diving into the project, I need to define steps to steps. But when I reach the `block-std` package. I don't actually understand what's package is! Is it a helper for the `block` and `lit` packages? Or have something more 🤔

Thank you for reading this! I love this product 🌸

**Comment by doodlewind** [comment]
Sorry for the delay, we've recently updated the project overview and arch charts:

> `@blocksuite/block-std`: A standard library required for modeling complete UI editable blocks. Its capabilities cover the structure of block fields, events, selection, clipboard support, etc. Note that this package does not directly define UI components for blocks but defines an abstract layer for mapping the block tree to any web framework's component tree.

![](https://github.com/toeverything/blocksuite/raw/master/packages/docs/images/package-overview.png)

Also, you can checkout the API walkthrough about `block-std` starting from [BlockSpec](https://blocksuite.io/block-spec.html).

Hope this could help!
